bitmap-team-deuces/lib/parse.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
```

Comment:
Great job!  I really like how you implemented the error checking for an undefined buffer.